buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -12,13 +12,24 @@ object Deps {
             val appCompat = AndroidLibrary(
                 name = "androidx.appcompat:appcompat:${Versions.Libs.Android.appCompat}"
             )
+            val coroutines = AndroidLibrary(
+                name = "org.jetbrains.kotlinx:kotlinx-coroutines-android:${Versions.Libs.MultiPlatform.coroutines}"
+            )
```

Comment:
why coroutines duplicated in android and multiplatform?

Reply:
Will be removed after adding the moko-mvvm.